Gateway/Http/Client/TransactionPosCloudSync.php

```diff
@@ -17,11 +17,10 @@
 use Adyen\Payment\Helper\Config;
 use Adyen\Payment\Helper\Data;
 use Adyen\Payment\Logger\AdyenLogger;
-use Magento\Payment\Gateway\Http\ClientInterface;
 use Magento\Payment\Gateway\Http\TransferInterface;
 use Magento\Store\Model\StoreManagerInterface;
 
-class TransactionPosCloudSync implements ClientInterface
+class TransactionPosCloudSync extends BaseTransaction
```

Comment:
Abstract `requestHeaders()` method has not been called in this class. Do we still need to extend `BaseTransaction`?